samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -113,6 +113,12 @@
   public static final String CLUSTER_MANAGER_SLEEP_MS = "cluster-manager.jobcoordinator.sleep.interval.ms";
   private static final int DEFAULT_CLUSTER_MANAGER_SLEEP_MS = 1000;
 
+  /**
+   * The container placement handler sleeps for a configurable time before checking for now ContainerPlacement messages
+   */
+  public static final String CLUSTER_MANAGER_CONTAINER_PLACEMENT_HANDLER_SLEEP_MS = "cluster-manager.container-placement-handler.sleep.interval.ms";
+  private static final int DEFAULT_CLUSTER_MANAGER_CONTAINER_PLACEMENT_HANDLER_SLEEP_MS = 5000;
```

Comment:
Do we expect different jobs to tune this timeout differently?
If not, i'd suggest removing this config

Reply:
No for the first pass we can read defaults!